tests/periph_i2c/main.c

```diff
@@ -448,27 +404,24 @@ int cmd_i2c_write_regs(int argc, char **argv)
     uint16_t addr;
     uint16_t reg;
     uint8_t flags = 0;
-    int len = argc - 4;
+    int len = argc - 5;
     uint8_t data[BUFSIZE];
+    int dev;
 
-    if (check_dev() == 1) {
-        return 1;
-    }
-
-    if (argc < 5) {
-        INVALID_ARGS;
-        printf("Usage:\n%s ADDR REG FLAG BYTE0 [BYTE1 ...]\n", argv[0]);
+    dev = _check_param(argc, argv, 5, 5 + BUFSIZE,
```

Comment:
shouldn't this `_check_param(argc, argv, 6, (5 + BUFSIZE),` because the `6` param contains the first byte, ie. the minimum. At least the help text suggests that the first byte is not optional, or not?

Reply:
ah no it should be `_check_param(argc, argv, 5, (5 + BUFSIZE -1)`, bc the last param is already the first byte hence buffer must be one less, right?